src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
 // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const newLeft = Math.round(wall.offsetWidth - spider.clientWidth) / 2;
+const newTop = Math.round(wall.offsetHeight - spider.clientHeight) / 2;
```

Comment:
Similar to the previous line, ensure that the division by 2 is applied to the entire expression. Use parentheses to ensure the correct calculation of newTop.